sdcm/cluster_k8s/gke.py

```diff
@@ -366,26 +367,28 @@ def terminate_k8s_node(self):
             Scylla Pod      X
             Scylla node     X
             '''))
+        node_name = self.node_name
         super().terminate_k8s_node()
 
         # Removing GKE instance and adding one node back to the cluster
         # TBD: Remove below lines when https://issuetracker.google.com/issues/178302655 is fixed
-
-        self.parent_cluster.node_pool.remove_instance(instance_name=self.node_name)
+        self.parent_cluster.node_pool.remove_instance(instance_name=node_name)
         self.parent_cluster.k8s_cluster.resize_node_pool(
-            self.parent_cluster.pool_name,
-            self.parent_cluster.node_pool.num_nodes
+            self.parent_cluster.node_pool.name,
+            self.parent_cluster.node_pool.num_nodes,
         )
 
+        self.wait_for_k8s_node_readiness()
```

Comment:
On termination operations we usually do not wait when target node comes up.
Two reason for that:
1. You might want not to wait till node go up back, for example you would want to decommission it right after
2. Other methods do not do that and we should be consistent across these methods

Reply:
It is the same approach which we have in EKS.

Reply:
It is a mistake we should not do that, we can wait till node is up, but not for a pod, on both EKS and GKE

Reply:
Ok, will remove it. But also will need to retest new approach to make sure there are no problems appear after it.